tests/_fixtures.py

```diff
@@ -83,6 +91,8 @@
     ) PRIMARY KEY (CartId);
 """
 
+# TODO: Add Proto Bundle DDL statement in EMULATOR_DDL once b/338557401
```

Comment:
Remove this TODO

Reply:
I would prefer to have this TODO as this is in the test suite and not in the client code. This will help any developer understand the changes needed when emulator tests are un-skipped thus saving time.

Reply:
Removed TODO